eventsocket/constants.go

```diff
@@ -162,3 +162,4 @@ var MapKeyIndex = map[string]int{
 }
 
 const FsEventMapSize = 100
+const customHeaderPrefix = "Variable_sip_h_x"
```

Comment:
is it possible to add this to the above block, maybe just to keep it consistent.